package.json

```diff
@@ -1,6 +1,6 @@
 {
   "engines": {
-    "node": "^20"
+    "node": "^20.10.0"
```

Comment:
suggests node lts, and runs on node 20 on vercel, otherwise it defaults to node 18